azkaban-exec-server/src/main/java/azkaban/execapp/FlowRunnerManager.java

```diff
@@ -427,6 +431,16 @@ private void cleanOlderProjects() {
     }
   }
 
+  public void deleteDirectory(ProjectVersion pv) throws IOException {
+    synchronized (pv) {
+      logger.warn("Deleting project: " + pv);
```

Comment:
`warn` is a stronger event that info. Deleting a project directory is an important event. Hence the categorization.